src/SDKs/CognitiveServices/dataPlane/AnomalyDetector/AnomalyDetector/Microsoft.Azure.CognitiveServices.AnomalyDetector.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.AnomalyDetector</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services AnomalyDetector APIs.</Description>
+    <Version>1.0.0-preview</Version>
```

Comment:
If you intend to start with a preview release, please start the versioning from `0.8.0-preview`

Reply:
If there some reason to do so? I have not seen any document/rule about version. You can give me some link if it exists so that I can go through the doc.

Reply:
This is the link for semantic versioning in general. For brand new preview Nuget packages, we have a guideline for versioning the packages starting (0.8.0-preview)